MobiFlight/Joysticks/VKB/VKBDevice.cs

```diff
@@ -0,0 +1,192 @@
+﻿using HidSharp;
+using HidSharp.Reports;
+using HidSharp.Reports.Input;
+using System;
+using System.Collections.Generic;
+
+namespace MobiFlight.Joysticks.VKB
+{
+    internal class VKBDevice : Joystick
+    {
+        private HidStream Stream;
+        private readonly HidDevice Device;
+        private readonly JoystickDefinition Definition;
+        private readonly VKBLedContainer Lights = new VKBLedContainer();
+        private HidDeviceInputReceiver InputReceiver;
+        private readonly byte[] InputReportBuffer = new byte[64];
+        private readonly SortedList<byte, VKBEncoder> Encoders = new SortedList<byte, VKBEncoder>();
+        int lastSeqNo = -1;
+
+        public VKBDevice(SharpDX.DirectInput.Joystick joystick, JoystickDefinition definition) : base(joystick, definition)
+        {
+            Definition = definition;
+            if (Device == null)
+            {
+                Device = GetMatchingHidDevice(joystick);
+                if (Device == null) return;
```

Comment:
Is this necessary? There's no code that follows so this really doesn't do anything.

If it does stick around, please put the return in { } to stay consistent with how we handle if statements elsewhere.

Reply:
Apparently any code that used it got refactored out before I made my first commit in my fork. Will clean the not-reakky-early return up.